### PR TITLE
new feature - container stats

### PR DESCRIPTION
**Pull request by navidys**
container_stats_dialog

Signed-off-by: Navid Yaghoobi <n.yaghoobi.s@gmail.com>

### CODE REVIEW

File: pdcs/containers/stats.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package containers
+
+import (
+	"github.com/containers/podman-tui/pdcs/connection"
+	"github.com/containers/podman/v3/pkg/bindings/containers"
```
Comment (vrothberg) [inline pdcs/containers/stats.go:5]: A mere drive-by-comment: at some point, we may need to bump the imported Podman to `v4` so the tui can talk to Podman 4.0+

Comment (navidys) [reply pdcs/containers/stats.go:5]: You are right.
I was thinking to add support for podman machine and remote features and then go to v4, but maybe better to bump it to v4 first since podman 4.0 is out.

**Comment by rhatdan** [comment]
LGTM
